compiler/quilt/tools/util.py

```diff
@@ -101,3 +106,31 @@ def sub_files(path, invisible=False):
         files = [x for x in files if not x.startswith('.')]
 
     return files
+
+
+def is_identifier(string):
+    """Check if string could be a valid python identifier
+
+    :param string: string to be tested
+    :returns: True if string can be a python identifier, False otherwise
+    :rtype: bool
+    """
+    val = re.match(r'^[a-zA-Z_]\w*$', string) and not keyword.iskeyword(string)
```

Comment:
Make the regex a global, like the old one - just to avoid recompiling it unnecessarily

Reply:
It's cached internally by python, and doesn't get recompiled.  But I can still make it global to avoid the cache lookup.